tests/lib/tools/tests.invariant

```diff
@@ -122,6 +123,22 @@ check_leftover_defer_sh() {
 	fi
 }
 
+check_broken_snaps() {
+	n="$1" # invariant name
+	(
+		# awk is hard - this just gives us the names of snaps that are broken
+		for brokesn in $(snap list --all | tail -n +2 | awk '{print $(NF) " " $1}' | grep 'broken ' | awk '{print $(NF)}' | sort | uniq); do
+			# could be multiple broken revisions
+			echo "snap $brokesn is broken"
```

Comment:
```suggestion
			echo "snap $broken is broken"
```

Reply:
ha I actually meant this to be a short-hand for "broken snap" but I can see how that's confusing, I will change it